symfony/plugins/orangehrmAdminPlugin/Api/ValidationUserNameAPI.php

```diff
@@ -75,6 +91,10 @@ public function getValidationRuleForGetOne(): ParamRuleCollection
                 self::PARAMETER_USER_NAME,
                 new Rule(Rules::STRING_TYPE),
                 new Rule(Rules::LENGTH, [null, self::PARAM_RULE_USER_NAME_MAX_LENGTH]),
+            ),
+            new ParamRule(
+                self::PARAMETER_USER_Id,
+                new Rule(Rules::STRING_TYPE),
```

Comment:
Should correct as `new Rule(Rules::POSITIVE),`

Reply:
done